### ISSUE TITLE
[MSAuth] Caused by HTTP 404 Not Found error.

### ISSUE THREAD

**Issue by notYEEET** [tropicbliss/buckshot#13]
```
Error: Failed to authenticate Microsoft account

Caused by:
    HTTP 404 Not Found
```

Very, very likely that I missed something so I'm sorry if something obvious went over my head.
Config below:

```
[account]
email = "myemail@gmail.com"
password = "password"
# Leave the rest empty if you do not have security questions
sq1 = ""
sq2 = ""
sq3 = ""
[config]
offset = 0
auto_offset = false
# Spread (if you are unsure leave it as it is)
spread = 0
microsoft_auth = true
gc_snipe = true
change_skin = false
skin_model = "slim"
skin_filename = "example.png"
# Name queueing (allows you to snipe multiple names sequentially)
# Note: This is an optional feature, leave it as it is if you only want to snipe one name
# Example:
# name_queue = ["Marc", "Dream"]
name_queue = []
```

**Comment by tropicbliss** [comment]
Nah, you did nothing wrong. I closed my authentication server to avoid paying so much for hosting costs. Right now in the new version, the authentication code is in the sniper itself to provide more transparency. However it is in beta. But it is pretty much stable right now, just need to update my user guide.
Download the newest version here (if you are using windows get the one with the .exe, the other one is for Linux):
https://github.com/tropicbliss/buckshot/releases/tag/v4.0.0
This version is incompatible with the older config.toml so to save you some time I've made a new config.toml according to the one you provided above (don't worry if it seems shorter, I've made the other options optional):

spread = 0
microsoft_auth = true
gc_snipe = true

[[account_entry]]
email = ""
password = ""

Also just a quick note here, your offset is most probably not 0, so with this config file an estimated offset should be calculated for you.
